starter-code/style.css

```diff
@@ -0,0 +1,93 @@
+
```

Comment:
Many thing of this css could be done in bootstrap. If you wanted to do it like this, it's ok!